todolist/settings.py

```diff
@@ -63,11 +63,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': '172.17.0.2',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'NAME': os.getenv('DATABASE_NAME', 'app_db'),
+        'USER': os.getenv('DATABASE_USER', 'app_user'),
+        'PASSWORD': os.getenv('DATABASE_PASSWORD', '1234'),
```

Comment:
The `DATABASE_PASSWORD` is hardcoded as '1234'. It's advisable to use an environment variable to manage this sensitive information securely.